src/common/SPELLS_SHAMAN.js

```diff
@@ -221,11 +227,13 @@ export default {
     id: 60103,
     name: 'Lava Lash',
     icon: 'ability_shaman_lavalash',
+    maelstrom: 30,
   },
   FURY_OF_AIR: {
     id: 197211,
     name: 'Fury of Air',
     icon: 'ability_ironmaidens_swirlingvortex',
+    maelstrom: 3,
   },
   EARTHEN_SPIKE: {
```

Comment:
Doesn't this cost Mailstrom as well? Also I noticed EARTH_SPIKE is a talent, so this can be omitted and `EARTHEN_SPIKE_TALENT` can be used instead. I just updated the talents parser to also extract mailstrom costs, so that should be in there too.

Reply:
There may be one or two more spells, but updated Earthen Spike and Fury of Air to use version in talents and removed the unneeded version in SPELLS_SHAMAN